spec/system/admin_company_value_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+require 'factory_bot_rails'
+
+RSpec.describe 'Modifying company values', type: :system do
```

Comment:
You are not only modifying ;) also creating deleting etc :)
```suggestion
RSpec.describe 'Admin Company Values CDUD', type: :system do
```